runtime/v2/shim/util.go

```diff
@@ -95,6 +95,7 @@ func BinaryName(runtime string) string {
 
 // BinaryPath returns the full path for the shim binary from the runtime name,
 // empty string returns means runtime name is invalid
+// TODO(2.0): Remove function
```

Comment:
question: Why is the function not removed? I couldnt find any usages of `BinaryPath`. Is this being imported and used in some other projects?

Reply:
I see that many functions in runtime are planned to be removed in 2.0, can we remove an exported function at will, even though it is not used in the containerd repo?
I'm not sure.